serializers/web/github_runner.rb

```diff
@@ -20,9 +20,9 @@ def self.base(runner)
       } : nil,
       vm_state: if runner.vm
                   runner.vm.display_state
-                elsif runner.strand.label == "wait_vm_destroy"
+                elsif runner.strand&.label == "wait_vm_destroy"
```

Comment:
It's a bit smelly to have multiple "safe navigation" like this. Should `runner` learn a `display_state`, or any other refactoring you can think of? Or is the cure worse than the disease?

A local variable might also be fine.

Reply:
Makes sense. I focused too much on these lines.

I refactored now. With recent changes, this state isn't only the state of virtual machine. So I moved to the runner model as "display_state" similar to the other models.